quarkus/runtime/src/test/java/org/keycloak/provider/quarkus/ConfigurationTest.java

```diff
@@ -110,57 +111,57 @@ public void testCamelCase() {
-        System.setProperty("kc.spi.hostname.default.frontend-url", "http://propvar.com");
-        assertEquals("http://propvar.com", initConfig("hostname", "default").get("frontendUrl"));
+        putEnvVar("KC_SPI_HOSTNAME_DEFAULT_FRONTEND_URL", "http://envvar.unittest");
+        System.setProperty("kc.spi.hostname.default.frontend-url", "http://propvar.unittest");
+        assertEquals("http://propvar.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testCLIPriorityOverSysProp() {
-        System.setProperty("kc.spi.hostname.default.frontend-url", "http://propvar.com");
-        System.setProperty(CLI_ARGS, "--spi-hostname-default-frontend-url=http://cli.com");
-        assertEquals("http://cli.com", initConfig("hostname", "default").get("frontendUrl"));
+        System.setProperty("kc.spi.hostname.default.frontend-url", "http://propvar.unittest");
+        System.setProperty(CLI_ARGS, "--spi-hostname-default-frontend-url=http://cli.unittest");
+        assertEquals("http://cli.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testDefaultValueFromProperty() {
-        System.setProperty("keycloak.frontendUrl", "http://defaultvalueprop.com");
-        assertEquals("http://defaultvalueprop.com", initConfig("hostname", "default").get("frontendUrl"));
+        System.setProperty("keycloak.frontendUrl", "http://defaultvalueprop.unittest");
+        assertEquals("http://defaultvalueprop.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testDefaultValue() {
-        assertEquals("http://filepropdefault.com", initConfig("hostname", "default").get("frontendUrl"));
+        assertEquals("http://filepropdefault.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testKeycloakProfilePropertySubstitution() {
         System.setProperty("kc.profile", "user-profile");
-        assertEquals("http://filepropprofile.com", initConfig("hostname", "default").get("frontendUrl"));
+        assertEquals("http://filepropprofile.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testQuarkusProfilePropertyStillWorks() {
         System.setProperty("quarkus.profile", "user-profile");
-        assertEquals("http://filepropprofile.com", initConfig("hostname", "default").get("frontendUrl"));
+        assertEquals("http://filepropprofile.unittest", initConfig("hostname", "default").get("frontendUrl"));
     }
 
     @Test
     public void testCommandLineArguments() {
-        System.setProperty(CLI_ARGS, "--spi-hostname-default-frontend-url=http://fromargs.com,--no-ssl");
-        assertEquals("http://fromargs.com", initConfig("hostname", "default").get("frontendUrl"));
+        System.setProperty(CLI_ARGS, "--spi-hostname-default-frontend-url=http://fromargs.unittest" + ARG_SEPARATOR + "--no-ssl");
```

Comment:
@pedroigor what should the --no-ssl do? Actually there is no such option and the build complains when running e.g. `java -jar server/target/lib/quarkus-run.jar build --spi-hostname-default-frontend-url=http://fromargs.unittest --no-ssl`. So just to make sure: Is this just "test foo", or is it sth that should actually work?